patches/chrome-browser-profiles-renderer_updater.cc.patch

```diff
@@ -11,6 +11,11 @@ index ce50c9ee3f9290b50b4a5704e8eec9dc91db3082..e6eed2042f34f13473ae4c71a44d31fe
 +      base::Value::FromUniquePtrValue(
```

Comment:
I think we can get rid of this entire patch because it doesn't actually do anything useful cc @yrliou

Reply:
Do we not need these default rules in the guest view? I think we need to double check the guest view case that things won't be wrongly blocked due to fingerprinting check for example without this default allow all rule, and adjust out logic if they would be. cc @pilgrim-brave

Reply:
I guess it's not really clear to me what we should do in a webview so maybe for now we should just convert this to a `define` patch and follow-up later